uui-core/src/types/tables.ts

```diff
@@ -147,6 +147,8 @@ export interface DataTableHeaderCellProps<TItem = any, TId = any> extends IEdita
     onSort(dir: SortDirection): void;
     onDrop?(params: DropParams<DataColumnProps<TItem, TId>, DataColumnProps<TItem, TId>>): void;
     renderFilter?: (dropdownProps: IDropdownBodyProps) => React.ReactNode;
+    /** Defines table columns gap size */
+    columnsGap?: '12' | '24';
```

Comment:
Add this prop to the DataTableCellMods interface in uui package, since DataRowProps it's a theme agnostic core interface, which shouldn't include mods and sizes.
Also add mention about '@default' value(related to the other interfaces too)

Reply:
done